lib/LightPanels/LightPanel.less

```diff
@@ -22,12 +22,8 @@
 	// Adding the content property here, instead of above, means that this element will only appear in
 	// the DOM during ".transitioning". This was altered to improve marquee animation performance.
 	// Pseudo-elements without a content property cannot receive styling.
-	&.transitioning {
-		-webkit-transform: translateZ(0);
-		transform: translateZ(0);
-
-		&:after {
-			content: '';
-		}
+	.moon-light-panels.transitioning &:after {
```

Comment:
I think this looks a little awkward, but wanted to take advantage of the parent kind having this class already, and it having the same meaning in that context. Without using `moon-light-panels`, I was afraid this would not be specific enough.